docs/reference/commandline/dockerd.md

```diff
@@ -986,6 +986,11 @@ If you have a group that doesn't match the username, you may provide the `gid`
 or group name as well; otherwise the username will be used as the group name
 when querying the system for the subordinate group ID range.
 
+To validate whether a docker daemon is running with user namespaces enabled or
+not, the output of `docker info` has been expanded. If the daemon is configured
```

Comment:
We don't have to explain that "it has been expanded" (since what?); docs reflect current state of the code, not a changelog :smile: